packages/fontconfig.rb

```diff
@@ -38,7 +39,7 @@ def self.build
       import sys
       sys.exit()
```

Comment:
Instead of calling the exit function manually, just left the file empty will be faster :)

Reply:
It's a hack to work around install issues. If I recall correctly this way we bypass getting errors that stop the install from succeeding.

Reply:
I do wish there was a better way. Feel free to try alternatives!

Reply:
Oh, mostly because this file is imported as a module so we need to exit manually